### PR TITLE
Added Codecov plugin to index

### PR DESCRIPTION
**Pull request by robertstettner**
This PR adds documentation for the following plugin: https://github.com/robertstettner/drone-codecov

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,4 @@ content/**.gen.*
 *.txt
 cmd
 server.go
+.idea/
```
Comment (appleboy) [inline .gitignore]: Add this to your global `.gitignore`

File: content/robertstettner/drone-codecov/index.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+image: robertstettner/drone-codecov
+---
+
+This plugin allows for pushing test coverage results to Codecov. The below pipeline configuration demonstrates simple usage:
+
+```yaml
+pipeline:
+  codecov:
+    image: robertstettner/drone-codecov
+    token: ${CODECOV_TOKEN}
```
Comment (appleboy) [inline content/robertstettner/drone-codecov/index.md]: You need to add `secret` section like that http://plugins.drone.io/appleboy/drone-gitlab-ci/

Comment (appleboy) [inline content/robertstettner/drone-codecov/index.md]: Don't use `${CODECOV_TOKEN}` anymore for the latest drone version.

**Comment by bradrydzewski** [comment]
let's use this svg instead https://s3-us-west-2.amazonaws.com/svgporn.com/logos/codecov.svg

**Comment by robertstettner** [comment]
All ready to go.

**Comment by bradrydzewski** [comment]
thanks!

**Pull request merged**